src/parse.c

```diff
@@ -1776,15 +1840,15 @@ static const mapping_entry_handler dhcp6_overrides_handlers[] = {
     {"routes", YAML_SEQUENCE_NODE, handle_routes},                                            \
     {"routing-policy", YAML_SEQUENCE_NODE, handle_ip_rules}
 
-#define COMMON_BACKEND_HANDLERS							\
+#define COMMON_BACKEND_HANDLERS							                                      \
```

Comment:
Completely minor: I guess the whitespace changes here are not needed?

Reply:
Definitely not needed, but I liked to clean it up on the way when I stumbled upon it.